packages/cli/src/commands/graph/command.ts

```diff
@@ -40,6 +40,7 @@ graphCommand
         rootPath: options.rootPath,
         ignore: options.ignore,
         externals: options.externals,
+        logGraph: true,
```

Comment:
NITl something more explicit here
```suggestion
        outputGraphToConsole: true,
```